docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -373,6 +373,42 @@ To enable a custom converter to handle `null` for a reference or value type, ove
 :::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/CustomConverterHandleNull.cs" highlight="18":::
 ::: zone-end
 
+## Preserve references
+
+::: zone pivot="dotnet-5-0"
+
+By default, reference data is only cached for each call to <xref:System.Text.Json.JsonSerializer.Serialize%2A> or <xref:System.Text.Json.JsonSerializer.Deserialize%2A>. To persist references from one `Serialize`/`Deserialize` call to another one, root the <xref:System.Text.Json.Serialization.ReferenceResolver> instance in the call site of `Serialize`/`Deserialize`. The following code shows an example for this scenario:
```

Comment:
Since this example covers serialization, maybe mention somewhere in this section that a similar approach can be adopted for deserialization.